src/components/Main/_Main.scss

```diff
@@ -0,0 +1,26 @@
+.Main {
+  display: grid;
+  grid-template-columns: 20% auto 20%;
```

Comment:
Best way to get the bottom grid-row without empty divs is to:
`grid-template-area: "temp-score temp-board temp-score" ". temp-input ."`